Makefile

```diff
@@ -1,39 +1,91 @@
-# Image URL to use all building/pushing image targets
```

Comment:
this is why I removed this repo from giantswarm/github - it's going to trash the Makefile and dependabot config :(

Reply:
see #10